FizzBuzzApp/Program.cs

```diff
@@ -1,3 +1,24 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+for (int i = 1; i < 101; i++) {
+    string answer = "";
+    
+    if (i % 3 == 0) { 
+        answer = "Fizz";

```

Comment:
Nitpick: Directly setting answer to Fizz here isn't consistent with the rest of the rules you have in place (which all append). Whilst in this case its not really an issue, if for example, someone wanted to add another rule that adds "Bing" on a multiple of two _before_ adding Fizz, then they would need to alter this rule to be `+=` instead. Given that it can be `+=` from the start, it would be slightly better to do that (and remain consistent with the other rules) rather than having something behave different here

As I said, not really an issue in this case but in more complex systems, making your functions as consistent and clean as possible makes extending them a lot easier and makes unintended reactors and side effects less likely :D